networkx/algorithms/isomorphism/tests/test_isomorphvf2.py

```diff
@@ -133,12 +160,35 @@ def test_subgraph(self):
         subgraph = self.create_graph(head / "si2_b06_m200.A99")
         graph = self.create_graph(head / "si2_b06_m200.B99")
         gm = iso.GraphMatcher(graph, subgraph)
+        assert not gm.is_isomorphic()
         assert gm.subgraph_is_isomorphic()
-        # Just testing some cases
         assert gm.subgraph_is_monomorphic()
 
-    # There isn't a similar test implemented for subgraph monomorphism,
-    # feel free to create one.
+    def test_subgraph_monomorphic(self):
+        # A is the subgraph
+        # B is the full graph
+        head = importlib.resources.files("networkx.algorithms.isomorphism.tests")
+        subgraph = self.create_graph(head / "si2_b06_m200.A99")
+        graph = self.create_graph(head / "si2_b06_m200.B99")
+        subgraph.remove_edge(29, 28)
+        gm = iso.GraphMatcher(graph, subgraph)
+        assert not gm.is_isomorphic()
+        assert not gm.subgraph_is_isomorphic()
+        assert gm.subgraph_is_monomorphic()
+
+    def test_subgraph_not_monomorphic(self):
+        # A is the subgraph
+        # B is the full graph
+        head = importlib.resources.files("networkx.algorithms.isomorphism.tests")
+        subgraph = self.create_graph(head / "si2_b06_m200.A99")
+        graph = self.create_graph(head / "si2_b06_m200.B99")
+        gm = iso.GraphMatcher(graph, subgraph)
+
+        # remove edge to make sg no longer subgraph isomorphic (previous test)
+        subgraph.remove_edge(29, 28)
```

Comment:
These are nearly the same right? I believe it would be good to avoid the double load and just make them one slightly longer test.